common/network-common/src/main/java/org/apache/spark/network/server/OneForOneStreamManager.java

```diff
@@ -139,6 +153,32 @@ public void checkAuthorization(TransportClient client, long streamId) {
     }
   }
 
+  @Override
+  public void chunkSent(Object streamId) {
```

Comment:
I mean `chunkSent(long streamId)`, the caller side is responsible to covert stream id to long.